README.md

```diff
@@ -44,6 +44,7 @@ node, npm, or yarn installed on a machine where you develop with Bazel.
 
 ```bash
 $ bazel run :install
+$ yarn install
```

Comment:
`bazel run :install` will run yarn in the current directory, so these are equivalent, except that bazel has a pinned version of yarn while your local one might be different